Controllers/FamilyController.cs

```diff
@@ -222,11 +245,29 @@ [FromForm] FamilyUpdateViewModel family
             return RedirectToAction("Index", "error", new { route = "/profile" });
         }
 
-        // once we updated successfully, not now update the job with new generation times
+        // once we updated successfully, update the job with new generation times
         _jobService.CreateorUpdateEmailJob(updated.Data!.Id);
 
         // update hub spot contact
-        _hubSpotService.UpdateContact(updated.Data.Name, updated.Data.Email, updated.Data.UserStatus, updated.Data.PhoneNumber);
+        _hubSpotService.UpdateContact(
+            updated.Data.Name,
+            updated.Data.Email,
+            updated.Data.UserStatus,
+            updated.Data.PhoneNumber
+        );
+
+        // TODO: Allow user to update subscription status
+        // User when from free trial to a letter queue eligible status (extendedFreeTrial or Subscribed)
+        // if (
+        //     (original.UserStatus == UserStatus.FreeTrial)
+        //     && (
+        //         updatedFamily.UserStatus == UserStatus.ExtendedFreeTrial
+        //         || updatedFamily.UserStatus == UserStatus.Subscribed
+        //     )
+        // )
+        // {
+        //     _letterQueueService.PopulateLetterQueue(familyId);
+        // }
```

Comment:
The question here is, are we allowing people to move from free trial to extended free trial? if so, this commented section would be needed. but again, we won't need to do anything different letter-queue wise if they move from free-trial to subscription. We may wan't to send them another hand written letter if they move from extended-free-trial to sub

Reply:
To allow the move from free trial to extended wouldn't be too big a lift. we could copy paste the "add your address?" HTML code from create profile to their profile page. and we would need to implement some logic behind the scenes to make it work how we'd like

Reply:
I like the idea of adding the user to the queue if they use extended or subscribe, just so we have them on hand if we decide to send out a customized message for them. 

I think it makes sense to allow the user to go from `free trial` to `extended free trial` as long as they are still in the 2 week free trial window. I will make a separate ticket for that